applications/luci-app-example/root/usr/share/rpcd/ucode/example.uc

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env ucode
```

Comment:
Technically, if we're to use the shebang and be able to call this uc file externally, set the +x bit on it also.

Reply:
Do ucode files under rpcd actually need a shebang? I can only find [one example](https://github.com/openwrt/luci/blob/0f743ad0f29a56704be8d13fb6d52c3617a7918a/applications/luci-app-ddns/root/usr/share/rpcd/ucode/ddns.uc#L1) in luci-app-ddns which has it, but all ucode files outside of rpcd don't.